### ISSUE TITLE
time.sleep false negative!

### ISSUE THREAD

**Issue by socketpair** [PyCQA/pylint#7269]
### Bug description

from time import time

time.sleep(1)

### Command used

```shell
pylint qwe.py
```

### Pylint output

```shell
************* Module qwe
qwe.py:1:0: C0114: Missing module docstring (missing-module-docstring)

------------------------------------------------------------------
Your code has been rated at 5.00/10 (previous run: 5.00/10, +0.00)
```

### Expected behavior

Report obvious error.

### Pylint version

```shell
pylint 2.13.7
astroid 2.11.6
Python 3.10.5 (main, Jun  9 2022, 00:00:00) [GCC 12.1.1 20220507 (Red Hat 12.1.1-1)]
```

### OS / Environment

Fedra 36

**Comment by Pierre-Sassoulas** [comment]
Thank you for opening the issue I can reproduce on the latest main. This require investigation, and probably some knowledge in astroid.

**Comment by socketpair** [comment]
```
pylint 3.0.4
astroid 3.0.3
Python 3.12.2 (main, Feb 21 2024, 00:00:00) [GCC 13.2.1 20231205 (Red Hat 13.2.1-6)]
```

Still actual
